src/main/kotlin/reactor/kotlin/extra/math/MathFluxExtensions.kt

```diff
@@ -101,10 +348,72 @@ fun <T> Flux<T>.sum(mapper: (T) -> Number): Mono<Long>
  *
  * @param mapper a lambda converting values to [Number]
  * @author Simon Baslé
- * @since 3.1.1
+ * @since 1.0.0
```

Comment:
this one should actually be deprecated like the others (since it pre-dates the PR)

Reply:
all original methods are now deprecated